concordium-smart-contract-testing/src/invocation/impls.rs

```diff
@@ -1309,6 +1403,29 @@ impl<'a, 'b> EntrypointInvocationHandler<'a, 'b> {
     }
 }
 
+/// A pair of the signatures, and the data.
+type DeserializedSignatureAndData<'a> = (AccountSignatures, &'a [u8]);
+
+/// Return a triple of
+/// - the number of signatures
+/// - the signatures
+/// - the data
+fn deserial_signature_and_data_from_contract(
+    payload: &[u8],
+) -> anyhow::Result<DeserializedSignatureAndData> {
+    use concordium_base::contracts_common::{Deserial, Read};
```

Comment:
Normally I would, but Deserial is a bit sensitive since there are two of them, and it is critically important that we use the right one here, the one from contracts common. And it is only needed in this function.

Reply:
I added a comment.